core/proc/element_reference_spec.rb

```diff
@@ -14,3 +16,14 @@
 describe "Proc#call on a Proc created with Kernel#lambda or Kernel#proc" do
   it_behaves_like :proc_call_on_proc_or_lambda, :call
 end
+
+ruby_version_is "2.6" do
+  describe "Proc#[] with frozen_string_literals" do
```

Comment:
This should really be a ruby_bug guard, otherwise the spec won't be executed e.g. on JRuby until Ruby 2.6 but the behavior should be fixed before that.
I fixed it in b556b57ac91986319a96edbf96b987ea85472ac7